src/providers/ark-api/ark-api.ts

```diff
@@ -100,7 +100,7 @@ export class ArkApiProvider {
     });
   }
 
-  public fetchAllDelegates(): Observable<arkts.Delegate[]> {
+  public fetchDelegates(getAllDelegates: boolean, numberDelegatesToGet: number): Observable<arkts.Delegate[]> {
```

Comment:
In all instances, `getAllDelegates` is always false. I'd make it an optional parameter at the end instead

```
getAllDelegates: boolean = false
```